api/Controllers/ReposController.cs

```diff
@@ -44,7 +50,20 @@ public async Task<IActionResult> Get(int id)
     [HttpPost]
     public async Task<IActionResult> Post(GitRepoCreateDto repo)
     {
+        // Create Repo
         var gitRepository = await _gitRepoService.Create(repo.Adapt<GitRepo>());
+
+        // Parse all commits
+        var parsedCommits = await _parser.Parse(new Uri(repo.Url));
+        var adaptedCommits = parsedCommits.Adapt<List<GitCommit>>();
+        adaptedCommits.ForEach(item => item.GitRepoId = gitRepository.Id);
+
+        foreach (var commit in parsedCommits.Where(a => a.Hash.Equals("off")))
+        {
+            Log.Information("{@Commit}", commit);
```

Comment:
Alleen loggen? Verder doen we er niks mee?

Reply:
Dit is logica van @YVbakker

Reply:
was nog van het debuggen